include/router_device.hrl

```diff
@@ -1,3 +1,25 @@
+-record(device_v7, {
```

Comment:
This is a good opportunity to add `rx_delay` to device record, also.

Remember that `rx_delay` has multiple components: 
1. `metadata` contains user-supplied value from Console API
2. but we also *abuse* `metadata` for tracking state: 
    + Previously established value
    + plus whether request has been sent via downlink but not yet ACK'd by device